pkg/cmd/controller/image.go

```diff
@@ -166,7 +166,7 @@ func RunImageImportController(ctx *ControllerContext) (bool, error) {
 		ctx.ClientBuilder.OpenshiftImageClientOrDie(infraImageImportControllerServiceAccountName),
 		informer,
 	)
-	go controller.Run(5, ctx.Stop)
+	go controller.Run(100, ctx.Stop)
```

Comment:
50?  Determined from pending queue?

Reply:
100 is just a random round number, I can put 50, if you want. We also can spawn a go-routine for every incoming event from API server and have only one reader from the queue. It depends on the purpose of this limit, what does it protect?

Reply:
It protects us from blowing up from one very bad image stream - i would in general prefer to make progress in pathological cases vs degrade.  timeouts in client code are ok as bounds, but the primary purpose of this queue is to ensure *eventually* something is imported.  not every import has a hard SLA of 15m for example.